diablo/jobs/instructor_emails_job.py

```diff
@@ -69,22 +66,28 @@ def email_scheduled_courses(self):
                     email_template = EmailTemplate.get_template_by_type(template_type)
                     if email_template:
                         for instructor in course['instructors']:
-                            BConnected().send(
-                                message=interpolate_email_content(
-                                    templated_string=email_template.message,
-                                    course=course,
-                                    instructor_name=instructor['name'],
-                                    recipient_name=instructor['name'],
-                                    recording_type_name=scheduled.recording_type,
-                                ),
-                                recipients=course['instructors'],
```

Comment:
This might be one of the duplicate-email bugs we've been seeing; for every instructor it appears to send to all instructors.

Reply:
Good catch. All looks good. Waiting on Travis